src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionUtilityToolFilter.java

```diff
@@ -130,10 +132,11 @@ public void doFilter(ServletRequest request, ServletResponse response,
 			chain.doFilter(useRequest, response);
 		}
 		else if(session.getAttribute(LAUNCH_TYPE).equals("sc")){
-
-			if(!checkForAuthorization(useRequest)) {
-				useResponse.sendError(403);
-				return;
+			if(!useRequest.getPathInfo().equals(STATUS_PAGE)  && !useRequest.getPathInfo().equals(PING_PAGE)){
```

Comment:
Consider checking status and authentication conditions in a single if rather than nesting if statements.  Since the nested if is so simple it may be clearer to make it into a single test.